src/server/world.zig

```diff
@@ -411,6 +411,120 @@ const WorldIO = struct { // MARK: WorldIO
 	}
 };
 
+pub const BlockDamage = struct { // MARK: BlockDamage
```

Comment:
Could you merge the two versions in a generic way?

Reply:
Maybe, but why do you want to merge them? Literally everything else that has client and server side is coded separate if both sides are not identical, and here they are not. Where should this merged version be placed? How should it be merged?

Reply:
I want to merge them, so I don't have to review how these two versions are different.
I just noticed they are using the same data structures, but if you say their implementation is sufficiently distinct (i.e. they are different beyond the fact that the client needs to add it to the mesh), then we can keep it like this.